cmd/adsysd/integration_tests/testdata/TestPolicyUpdate/golden/purge_policies_for_all_connected/dconf/profile/userintegrationtest@example.com

```diff
@@ -0,0 +1,3 @@
+user-db:user
```

Comment:
here, I think userintegrationtests@example.com is considered as not connected?

Reply:
ok, I think we need to redefine the empty state then for dconf (probably another PR on the side?) it should remove those files.